src/dashboard/src/components/mcp/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```

Comment:
`components.mcp` doesn't have any models or signals, and I don't think we'll ever add any. I think we can remove this one.

Reply:
Thank you for your suggestion.